tools/xtce_generation/telemetry_generator.py

```diff
@@ -132,8 +132,6 @@ def create_parameter_types(self, parameter_type_set, unique_lengths):
                 )
                 binary_parameter_type.attrib["name"] = parameter_type_ref_name
 
-                Et.SubElement(binary_parameter_type, "xtce:UnitSet")
```

Comment:
I don't think this is necessary? It just created one line for me:
`<xtce:UnitSet />` and it isn't in any of the other type parameter generator flows. So I just removed it. Looks like it dates back to the original pass of this file, so I suspect it was just something extra that fell through the cracks when we added the byte/sint/uint flow.